src/chronometer-centiseconds.js

```diff
@@ -1,37 +1,51 @@
 class Chronometer {
-  constructor() {
-    // ... your code goes here
-  }
+  currentTime = 0;
```

Comment:
haha nice :)

Reply:
who needs constructors anyways!

Reply:
Oh, yes... it passed, so I didn´t notice